tests/test_sidebar.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+
+@pytest.mark.smoke
+@pytest.mark.regression
+def test_logout(login_to_app):
+    sidebar = login_to_app
+    sidebar.click_logout()
+    assert sidebar.is_login_present(), 'Login sidebar item is not present'
```

Comment:
strange logic. 
Logout should send you to login page

Reply:
fixed. logout returns LoginPage. local import was used to avoid circular import.